spec/policies/group_policy_spec.rb

```diff
@@ -54,10 +54,6 @@
       it "permits group creation, viewing, and editing" do
         expect(policy).to permit_only_actions(%i[show new edit create update])
       end
-
-      it "does not allow add_editor or upgrade" do
-        expect(policy).to forbid_only_actions(%i[add_editor upgrade])
-      end
```

Comment:
Why was this test removed?

Reply:
We removed it as there is a test above that tests that only certain actions are permitted, so already tests this. We previously removed the other tests where there was both `permit_only_actions` and `forbid_only_actions` for the same context, so this was the only one left